stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -513,21 +489,18 @@ public struct ThrowingTaskGroup<ChildTaskResult: Sendable, Failure: Error> {
     _enqueueJobGlobal(Builtin.convertTaskToJob(childTask))
   }
 
-  /// Add a child task to the group.
-  ///
-  /// ### Error handling
-  /// Operations are allowed to `throw`, in which case the `try await next()`
-  /// invocation corresponding to the failed task will re-throw the given task.
+  /// Adds a child task to the group, unless the group has been canceled.
   ///
-  /// The `add` function will never (re-)throw errors from the `operation`.
-  /// Instead, the corresponding `next()` call will throw the error when necessary.
+  /// This method doesn't throw an error, even if the child task throws.
+  /// Instead, the next call to `TaskGroup.next()` rethrows that error.
```

Comment:
The "corresponding" was important; it is not the "next `next()` call", consider this:

```swift
{ g in 
group.spawn { sleep(10); return 111 }
group.spawn { throw Boom() }

try await group.next() // boom!
}
 ```

since "next" is in completion order; so each next completion corresponds to some spawn, but the order is "by completion order" rather than "by submission order"